### PR TITLE
better freetype version checking

### PR DESCRIPTION
**Pull request by prinsherbert**

### CODE REVIEW

**Comment by prinsherbert** [comment]
continuation of https://github.com/matplotlib/matplotlib/pull/6403, to fix the commit message log - as an alternative `git rebase`.

File: lib/matplotlib/__init__.py [hunk 1]
```diff
@@ -1570,7 +1570,15 @@ def _init_tests():
         warnings.warn(
             "matplotlib is not built with the correct FreeType version to run "
             "tests.  Set local_freetype=True in setup.cfg and rebuild. "
-            "Expect many image comparison failures below.")
+            "Expect many image comparison failures below. "
+            "Expected freetype version {0}. "
+            "Found freetype version {1}."
```
Comment (mdboom) [inline lib/matplotlib/__init__.py]: Space after period.

Comment (prinsherbert) [inline lib/matplotlib/__init__.py]: This was also not the case in the original, so I don't see the problem.

Comment (WeatherGod) [inline lib/matplotlib/__init__.py]: What was not the case in the original? The original had spaces between sentences. This one is missing a space between two sentences.

Comment (WeatherGod) [inline lib/matplotlib/__init__.py]: And, don't worry. We won't ask you to rebase/squash for this. It was only to clean out a messy revision history. Fixing one little thing doesn't need a rebase/squash in my opinion.

**Comment by WeatherGod** [comment]
All the appveyor jobs are still queued. I am going to go ahead and just merge this.

**Comment by WeatherGod** [comment]
backported to v2.x as ca5324c

**Comment by WeatherGod** [comment]
@prinsherbert, thank you for bearing with us through the review process.

**Comment by prinsherbert** [comment]
Thank you for merging!

**Pull request merged**